src/Transition/CreateNewIDFUsingRulesV9_5_0.f90

```diff
@@ -435,6 +438,93 @@ SUBROUTINE CreateNewIDFUsingRules(EndOfFile,DiffOnly,InLfn,AskForInput,InputFile
+                OutArgs(6)=OutArgs(8)
+                OutArgs(7:10)=InArgs(12:15)
+                OutArgs(11:22)=InArgs(18:29)
+                OutArgs(23:CurArgs-9)=InArgs(32:CurArgs)
+                CurArgs = CurArgs - 9
+                nodiff=.false.
+
+                CALL GetNewObjectDefInIDD('ZONEHVAC:LOWTEMPERATURERADIANT:CONSTANTFLOW:DESIGN',NumArgs,AorN,ReqFld,ObjMinFlds,FldNames,FldDefaults,FldUnits)
+                POutArgs(1) = OutArgs(2)
+                POutArgs(2:4) = InArgs(5:7)
+                POutArgs(5:7) = InArgs(9:11)
+                POutArgs(8:9) = InArgs(16:17)
+                POutArgs(10:11) = InArgs(30:31)
+                POutArgs(12) = InArgs(34)
+                CALL WriteOutIDFLines(DifLfn,'ZONEHVAC:LOWTEMPERATURERADIANT:CONSTANTFLOW:DESIGN',12,POutArgs,FldNames,FldUnits)
+                nodiff=.false.
+
+              CASE('ZONEHVAC:BASEBOARD:RADIANTCONVECTIVE:WATER')
+                CALL GetNewObjectDefInIDD(ObjectName,NwNumArgs,NwAorN,NwReqFld,NwObjMinFlds,NwFldNames,NwFldDefaults,NwFldUnits)
+                OutArgs(1)=InArgs(1)
+                OutArgs(2)=TRIM(InArgs(1))//(' Design Object')
+                OutArgs(3:7)=InArgs(2:6)
+                OutArgs(8)=InArgs(8)
+                OutArgs(9)=InArgs(11)
+                OutArgs(10:CurArgs-5)=InArgs(15:CurArgs)
+                CurArgs = CurArgs - 5
+                nodiff=.false.
+
+                CALL GetNewObjectDefInIDD('ZONEHVAC:BASEBOARD:RADIANTCONVECTIVE:WATER:DESIGN',NumArgs,AorN,ReqFld,ObjMinFlds,FldNames,FldDefaults,FldUnits)
+                POutArgs(1) = OutArgs(2)
+                POutArgs(2) = InArgs(7)
+                POutArgs(3:4) = InArgs(9:10)
+                POutArgs(5:7) = InArgs(12:14)
+                CALL WriteOutIDFLines(DifLfn,'ZONEHVAC:BASEBOARD:RADIANTCONVECTIVE:WATER:DESIGN',7,POutArgs,FldNames,FldUnits)
+                nodiff=.false.
+
+              CASE('ZONEHVAC:BASEBOARD:RADIANTCONVECTIVE:STEAM')
+                CALL GetNewObjectDefInIDD(ObjectName,NwNumArgs,NwAorN,NwReqFld,NwObjMinFlds,NwFldNames,NwFldDefaults,NwFldUnits)
+                OutArgs(1)=InArgs(1)
+                OutArgs(2)=TRIM(InArgs(1))//(' Design Object')
+                OutArgs(3:5)=InArgs(2:4)
+                OutArgs(6)=InArgs(6)
+                OutArgs(7:8)=InArgs(9:10)
+                OutArgs(9)=InArgs(11)
+                OutArgs(9:CurArgs-5)=InArgs(14:CurArgs)
+                CurArgs = CurArgs - 5
+                nodiff=.false.
+
+                CALL GetNewObjectDefInIDD('ZONEHVAC:BASEBOARD:RADIANTCONVECTIVE:STEAM:DESIGN',NumArgs,AorN,ReqFld,ObjMinFlds,FldNames,FldDefaults,FldUnits)
+                POutArgs(1) = OutArgs(2)
+                POutArgs(2) = InArgs(5)
+                POutArgs(3:4) = InArgs(7:8)
+                POutArgs(5:7) = InArgs(11:13)
+                CALL WriteOutIDFLines(DifLfn,'ZONEHVAC:BASEBOARD:RADIANTCONVECTIVE:STEAM:DESIGN',7,POutArgs,FldNames,FldUnits)
+                nodiff=.false.
```

Comment:
But after transitioning a baseboard file, I was able to run the file successfully with a clean error file.  I did not test every permutation of object, but I'll continue testing a couple more.